articles/multimodal-neurons-in-artificial-neural-networks/index.md

```diff
@@ -0,0 +1,97 @@
+
+In Biology, we expect neurons that respond not to specific individual words or features but abstract concepts. For example, biological neurons would respond to "Halle Berry" photos, drawings and sketches of Halle Berry, and texts of Halle Berry. In Artificial Neural Networks, we have not seen the concept of the multimodal neuron perception being used. We have only seen neurons responding to the same class of images because we train them as image classifiers. Besides, these models do not generalize well to other modalities such as sketches and texts.
+
+In the main paper, they present an overview of the different neurons that they find. They have gone through these neurons and have used their feature visualization technique previously used in their CLIP model, with every single one of them. 
+
+### Categories of Neurons
+
+A few examples of the neurons they found include:
+
+1. Region Neurons
+
+This type of neuron responds to different kinds of images related to a particular geographic region and cities. In the example below, the neuron responds to the USA's architecture, nature of Europe's landscape, faces of Indians, and text related to West Africans.
+
+![Region Neurons](/engineering-education\multimodal-neurons-in-artificial-neural-networks/region-neuron.PNG)
+
+*[Image Source: Distill](https://distill.pub/2021/multimodal-neurons/)*
+
+2. Person Neurons
+
+They found neurons that respond to the faces of specific persons. For example, the images below show content associated with Donald Trump, Lady Gaga, Ariana Grande, and Elvis Presley. Besides faces, these neurons could also respond to texts, architectures, poses, and logos associated with the same persons. 
+
+![Person Neuron](/engineering-education\multimodal-neurons-in-artificial-neural-networks/person-neuron.PNG)
+
+*[Image Source: Distill](https://distill.pub/2021/multimodal-neurons/)*
+
+3. Emotion Neurons
+
+They found neurons that respond to words, facial expressions, and any content associated with an emotional or mental state. These emotional states include sadness, surprise, shock, crying, happiness, and sleepiness. This example is shown below:
+
+![Emotion Neuron](/engineering-education\multimodal-neurons-in-artificial-neural-networks/emotion-neuron.PNG)
+
+*[Image Source: Distill](https://distill.pub/2021/multimodal-neurons/)*
+
+### Typographic Attacks
+
+1. The model responds heavily to rendered text. This can be seen from the adversarial attacks where, i.e., take an apple and attach a sticker labeled iPod on it, it labels the picture as an iPod instead of an apple. This example shows that the text might still be too dominant in this model. They also show that randomly rendering texts on images confuse the model. For example, rendering texts of "pizza" on top of a dog image confuses the classifier by making it classify the picture as "pizza" instead of a "dog."

```

Comment:
This section feels a bit disjointed from the rest of the article. For instance, a reader may wonder what model is being referred to by "the model". As such, introduce the section in a few sentences to maintain the flow of the article.